include/osg/KdTree

```diff
@@ -212,7 +212,6 @@ class OSG_EXPORT KdTreeBuilder : public osg::NodeVisitor
     protected:
 
         virtual ~KdTreeBuilder() {}
-
```

Comment:
Can we go through and remove any changes that are whitespace only? No point in dragging them forward and cluttering the diffs between the master osg 3.6 branch and our makOsg branch.

Reply:
I wouldn't worry about the whitespace changes if there are other changes in the same file. Just when the only change in a file is whitespace only.